### PR TITLE
Update FreeNAS/TrueNAS guest installation steps.

### PR DESCRIPTION
**Pull request by etomm**
These are extended steps that should work for all version 11 until 12.0-U1.

The issue from where they started is #446

### CODE REVIEW

File: docs/guests.md [hunk 1]
```diff
@@ -126,36 +126,54 @@ For OpenBSD search [the forum](https://xcp-ng.org/forum). See for example [this
...
+   
+6. Once the package is installed, you need to tell FreeNAS to start the `xe-daemon` process when starting:
+   1. Go to _Tasks -> Init/Shutdown Script_
+   2. Create a new task with the following settings:
+      * Type: _Command_
+      * Command: `/usr/local/sbin/xe-daemon -p /var/run/xe-daemon.pid &`
+      * When: _Pre Init_
+      * Enabled: Checked
+
+7. Reboot. If you do not plan to reboot the VM, you can start the deamon manually running the command `/usr/local/sbin/xe-daemon -p /var/run/xe-daemon.pid &`. After you'll see a FreeBSD icon in your VM list on Xen Orchestra, and you can restart/shutdown the VM properly from the Web UI.
```
Comment (stormi) [inline docs/guests.md:174]: deamon => daemon

File: docs/guests.md [hunk 2]
```diff
@@ -126,36 +126,54 @@ For OpenBSD search [the forum](https://xcp-ng.org/forum). See for example [this
...
+
+More insights and options are available in [this issue](https://github.com/xcp-ng/xcp/issues/172#issuecomment-548181589) or [this issue](https://github.com/xcp-ng/xcp/issues/446).
```
Comment (stormi) [inline docs/guests.md:176]: Is it on purpose that the first link is to a specific comment in the issue?

Comment (etomm) [reply docs/guests.md:176]: It's like that in the current documentation, and I did not want to change it cause I did not write the previous docs. I wondered if in fact I should point even the second to a specific comment.

File: docs/guests.md (hunk 1 above)
Comment (etomm) [reply docs/guests.md]: Fixed

**Review by stormi — APPROVED** [review]

**Pull request merged**